protobuf/xain_fl/grpc/coordinator.proto

```diff
@@ -55,22 +55,15 @@ message StartTrainingReply {
 
 message EndTrainingRequest {
   ThetaUpdate theta_update = 1;
```

Comment:
this is quite general regarding every occurrence of the naming `theta`: in mathematical papers it definitely makes sense to use the letter θ (theta) as a variable for the weights. but since we have to write python variables in latin letters, we should therefor use expressive variable names like `weights`, etc.

Reply:
Agree. What do you think about `weights_local`? 
```suggestion
  WeightsLocal weights_local = 1;
```

Reply:
Since the weight update is also containing the number of samples for federated averaging, `ThetaUpdate` was renamed to `WeightUpdate`.